Remove obsolete fallback script now that picture element is supported

Anuj-Rathore24 opened the pull request:
## Summary
Remove Fallback Script which was used to replace webp images with jpeg alternatives on older browsers.
Fixes #996

## Relevant technical choices
- Remove the fallback File.
- Remove Function webpUploadsDetectFallback and the action associate with this.

Anuj-Rathore24 committed:
Refactor: Removed File fallback.js

Anuj-Rathore24 committed:
Refactor: Remove function and associated action.

Removed the webpUploadsDetectFallback function and the associated foreach loop that adds actions for this function.

westonruter commented:
Looks like this test case needs to be removed as well: `Test_WebP_Uploads_Load::test_it_should_add_fallback_script_if_content_has_updated_images`

Anuj-Rathore24 committed:
Tests: Remove Test Case
	- Remove Test case that checks that the fallback script is added when a post with updated images is rendered.

westonruter reviewed (approved):
Thank you!
